README.md

```diff
@@ -1,6 +1,86 @@
-# silk-paper
-Package for the automatic generation of REST documentation based on the OpenApi specification. Key name: Silk
+# silk-paper-node
+Package for the automatic generation of REST documentation based on the OpenApi specification.
+
+## Installing
+Install with `npm i @wolox/silk-paper`.
+
+## Usage
+* In `app.js`:
+We are using [`swagger-ui-express`](https://www.npmjs.com/package/swagger-ui-express) package for render the documentation, so first you have to install it.
+```
+const swaggerUi = require('swagger-ui-express');
+const silkPaper = require('@wolox/silk-paper');
+
+app.use('/docs', swaggerUi.serve, swaggerUi.setup(silkPaper.buildDocs()));
```

Comment:
Remember that the important thing to pass to this method is de directory of the docs, we should put it in the usage example